building/optee_with_rust.rst

```diff
@@ -0,0 +1,100 @@
+.. _optee_with_rust:
+
+################ 
+OP-TEE with Rust
+################
+
+This document describes how to build OP-TEE client and trusted applications
+written in `Rust <https://www.rust-lang.org>`_ with `Teaclave TrustZone SDK
+<https://github.com/apache/incubator-teaclave-trustzone-sdk>`_.
+
+Clone OP-TEE repo 
+*****************
+
+Currently, Teaclave TrustZone SDK is compatible with QEMUv8 (aarch64).
+
+Before building examples written with Teaclave TrustZone SDK, you should 
+clone the OP-TEE repo first. For QEMUv8, run:
+
+.. code-block:: bash
+
+    $ mkdir YOUR_OPTEE_DIR && cd YOUR_OPTEE_DIR
+    $ repo init -u https://github.com/OP-TEE/manifest.git -m qemu_v8.xml 
+    $ repo sync
+
+The source code of Teaclave TrustZone SDK is located in 
+``YOUR_OPTEE_DIR/optee_rust/`` containing a set of examples written in Rust
+using the SDK.
+
+For more information about building OP-TEE using QEMUv8, see `run OP-TEE using
+QEMU
+<https://optee.readthedocs.io/en/latest/building/devices/qemu.html#qemu-v8>`_ .
+
+Compile Rust examples 
+*********************
+
+To build Rust examples in ``optee_rust/examples/`` and install them in
```

Comment:
I find the sentence confusing: building *in* optee_rust/examples/? Rather build ~~*in*~~ optee_rust/examples/. I would rephrase to something like:
```
Rust examples applications are located in ``optee_rust/examples/``. To build and install them with Buildroot, run: